app/model/jobs/steps/ModifyContentTags.scala

```diff
@@ -0,0 +1,70 @@
+package model.jobs.steps
+
+import com.gu.tagmanagement.{OperationType, TagWithSection, TaggingOperation}
+import model.{BatchTagOperation, Section, Tag, TagAudit}
+import model.jobs.StepStatus
+
+import scala.concurrent.duration._
+import model.jobs.Step
+import play.api.Logger
+import services.KinesisStreams
+import repositories.{ContentAPI, TagAuditRepository}
+
+import scala.language.postfixOps
+
+// The 'op' argument of this case class should really be the BatchTagOperation type but serializing enums to JSON isn't very easy with the
+// current version of play and enumeratim
+case class ModifyContentTags(tag: Tag, section: Option[Section], contentIds: List[String], op: String, `type`: String = ModifyContentTags.`type`, var stepStatus: String = StepStatus.ready, var stepMessage: String = "Waiting", var attempts: Int = 0) extends Step {
+  override def process = {
+    contentIds foreach { contentPath =>
+      val taggingOperation = TaggingOperation(
+        operation = OperationType.valueOf(op.replace("-", "")).get,
```

Comment:
Is `.get` safe here?

Reply:
Should be, we pass the string via an enum that validates the input is one of `remove`, `add-to-top` or `add-to-bottom`. Sadly the thrift enums drop the dashes so we need the replace.